src/ol/proj/proj4.js

```diff
@@ -66,3 +87,72 @@ export function register(proj4) {
     }
   }
 }
+
+/**
+ * @param {number} code The EPSG code.
+ * @return {Promise<string>} The proj4 definition.
+ */
+let epsgLookup = async function (code) {
+  const response = await fetch(`https://epsg.io/${code}.proj4`);
```

Comment:
Sorry for the late comment on this closed PR, but the results here can be unexpected. See the following:

```
code = 1;
response = await fetch(`https://epsg.io/${code}.proj4`);
```

`response.ok` results in `true`

Also when using `code = undefined` for example.

Unfortunately epsg.io returns a `200` header code, even though the page content says `500: internal server error`. Example: https://epsg.io/1.proj4

Shall I file a separate bug report for this?

Reply:
Better even would be a PR with a fix :-)